tracer/test/Datadog.Trace.Security.IntegrationTests/ApiSecurity/AspNetCoreApiSecurity.cs

```diff
@@ -33,7 +33,6 @@ protected AspNetCoreApiSecurity(AspNetCoreTestFixture fixture, ITestOutputHelper
         Directory.CreateDirectory(LogDirectory);
         EnvironmentHelper.CustomEnvironmentVariables.Add(ConfigurationKeys.AppSec.Rules, Path.Combine("ApiSecurity", "ruleset-with-block.json"));
         // necessary as the developer middleware prevents the right blocking response
```

Comment:
nit: no longer relevant

```suggestion
```

Reply:
good catch removed in 6fd2f47c71a4adc4b6bd07a6277a030d23e0a6ce